### ISSUE TITLE
Menu not working on ios

### ISSUE THREAD

**Issue by nsantini** [xotahal/react-native-material-ui#40]
The Toolbar menu (as seen in the demo app) works on Android but not on IOS

![screen shot 2016-10-25 at 11 07 37 am](https://cloud.githubusercontent.com/assets/438741/19665811/61443f18-9aa3-11e6-9a47-c7bc4de08923.png)

**Comment by xotahal** [comment]
Thank you @nsantini for report ..

**Comment by nicolasbrugneaux** [comment]
Any progress on this issue?

**Comment by shkyang** [comment]
Any idea when or if this feature is coming? would prefer not to get into custom modal code :)

**Comment by ctgardner** [comment]
Perhaps implementing this function on iOS as a no-op would be a good interim solution? It would stop the app from crashing. Thoughts?

**Comment by FezVrasta** [comment]
Why don't you use this internally?

`react-native-material-menu`

**Comment by mezod** [comment]
any updates on this issue? :)

**Comment by Gazfay** [comment]
Hey, any updates?

**Comment by xotahal** [comment]
@janziemba It probably doesn't generate material design menu, does it? Anyway, there is probably no need to wrap RN's action sheet to our Menu component. If someone wants to use these Action Sheets it is easier to use it directly from RN ... At least I think :) We should rather fix this issue ...

**Comment by Gazfay** [comment]
Yes i can use custom component, but i need ability to set options menu, because i use search toolbar and this toolbar render search icon as last item on the right side and only "menu" prop can render some icon buttons as last item. I'm waiting for fix.

**Comment by jextrevor** [comment]
Honestly, I agree with @FezVrasta's solution, the menu library there is more "material" looking than the default Android menu anyway

**Comment by jextrevor** [comment]
I've created a working version of react-native-material-menu integrated into react-native-material-ui, using the existing syntax for RightElement. You can view it on my [fork](https://github.com/jextrevor/react-native-material-ui). If the authors of this project feel it is good enough, I will make a pull request.

**Comment by senthilmca90** [comment]
Hi, Any update on this issue?

**Comment by joaodematejr** [comment]
+1

**Comment by jextrevor** [comment]
I've opened up a pull request that updates the project to use react-native-material-menu.
